tests/rel.js

```diff
@@ -0,0 +1,18 @@
+const data = require('../data/relationships');
+const status = require('../data/status');
+const { ObjectId } = require('mongodb');
+const Category = require('../enums/categories');
+
+let test = 
+async function(){
+    let relationship1 = await data.getRelationshipById('625b029d6081b3b2bf2dc8d0');

```

Comment:
Hope You have tested the creation of Relationship which is the most important of all!

Reply:
Oh didn't mean to add this actually, but yeah I tested all the data functions in my relationships-data branch